### ISSUE TITLE
Update github with latest version?

### ISSUE THREAD

**Issue by HimalayaTucson** [hoviekeller/speech-to-text#1]
Hello hoviekeller,

Can you update your github with your latest code for your Chrome extension?

Thank you for your work!

**Comment by DjDeets** [comment]
I just cloned the repo, turned on developer mode on the chrome extensions page, and then loaded the cloned repo as an extension. The result was me realizing that the extension on here appears to be the same as the extension on the chrome webstore (at least in terms of version number). Then going on to compare the release dates, this once again appears to be the most up to date version of the source.

TL;DR: This is the latest code for the chrome extension, despite the version numbers not matching.
